.github/workflows/linux-build-and-test.yaml

```diff
@@ -65,10 +65,10 @@ jobs:
         tar --extract --file postgresql.tar.bz2 --directory ~/$PG_SRC_DIR --strip-components 1
         cd ~/$PG_SRC_DIR
         if [[ "${{ runner.os }}" == "Linux" ]]; then
-          ./configure --prefix=$HOME/$PG_INSTALL_DIR ${{ matrix.pg_build_args }} --with-llvm LLVM_CONFIG=${{ matrix.llvm_config }} --with-openssl --without-readline --without-zlib --without-libxml ${{ matrix.pg_extra_args }}
+          ./configure --prefix=$HOME/$PG_INSTALL_DIR ${{ matrix.pg_build_args }} --with-llvm LLVM_CONFIG=${{ matrix.llvm_config }} --with-openssl --enable-tap-tests --without-readline --without-zlib --without-libxml ${{ matrix.pg_extra_args }}
         else
           # the current github macos image has a buggy llvm installation so we build without llvm on mac
-          ./configure --prefix=$HOME/$PG_INSTALL_DIR ${{ matrix.pg_build_args }} --with-openssl --without-readline --without-zlib --without-libxml ${{ matrix.pg_extra_args }}
+          ./configure --prefix=$HOME/$PG_INSTALL_DIR ${{ matrix.pg_build_args }} --with-openssl --enable-tap-tests --without-readline --without-zlib --without-libxml ${{ matrix.pg_extra_args }}
```

Comment:
Is it really necessary to build PostgreSQL with `--enable-tap-tests` in order to make tap tests work? It might be really inconvenient for cases when PostgreSQL is installed using packages, if you already have a bunch of pre-built versions or you need to run tests against some specific binary.

Any idea what this tweak does to PostgreSQL?

Reply:
I have removed `--enable-tap-tests` now